src/main/java-templates/utils/ProcessorSolutionInfo.java

```diff
@@ -0,0 +1,19 @@
+package com.commercetools.paymenttoorderprocessor.utils;
+
+import io.sphere.sdk.client.SolutionInfo;
+
+
+
```

Comment:
Can you delete these extra lines?

Reply:
c86bd0caa8443f541300002e879526a8d3465d98